src/components/Table/TableHead.js

```diff
@@ -21,37 +21,66 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import classnames from 'classnames'
 import { isEmpty } from 'lodash'
+import classNames from 'classnames'
 
-import { Tip, Tooltip, TextTooltipTemplate } from 'igz-controls/components'
-
-const TableHead = React.forwardRef(({ content, mainRowItemsCount, selectedItem }, ref) => {
-  return (
-    <div className="table-head" ref={ref}>
-      {content.map((tableItem, index) => {
-        const cellClassNames = classnames(
-          'table-head__item',
-          tableItem.class,
-          !isEmpty(selectedItem) && index >= mainRowItemsCount && 'table-body__cell_hidden'
-        )
-
-        return tableItem.type !== 'hidden' && !tableItem.hidden ? (
-          <div className={cellClassNames} key={`${tableItem.header}${index}`}>
-            <Tooltip template={<TextTooltipTemplate text={tableItem.header} />}>
-              {tableItem.header}
-            </Tooltip>
-            {tableItem.tip && <Tip text={tableItem.tip} />}
-          </div>
-        ) : null
-      })}
-      <div className="table-body__cell action_cell" />
-    </div>
-  )
-})
+import { Button, Tip, Tooltip, TextTooltipTemplate } from 'igz-controls/components'
+
+const TableHead = React.forwardRef(
+  ({ content, mainRowItemsCount, selectedItem, sortProps }, ref) => {
+    const { selectedColumnName, getSortingIcon, sortTable } = sortProps
+
+    const getHeaderClasses = (tableItemClass, headerId) =>
+      classNames(
+        'table-head__item',
+        tableItemClass,
+        'table__header-item-sortable',
+        selectedColumnName === headerId && 'table__header-item-sortable-active'
+      )
+
+    return (
+      <div className="table-head" ref={ref}>
+        {content.map(({ headerId, isSortable, ...tableItem }, index) => {
+          const cellClassNames = classnames(
+            'table-head__item',
+            tableItem.class,
+            !isEmpty(selectedItem) && index >= mainRowItemsCount && 'table-body__cell_hidden'
+          )
```

Comment:
You can also create a function for these classnames. You can call it `getHeaderCellClasses`